app/services/data/get-capacity-breakdown.js

```diff
@@ -1,29 +1,45 @@
 const knex = require('../../../knex').web
 const orgUnitFinder = require('../helpers/org-unit-finder')
+const ORGANISATION_UNIT = require('../../constants/organisation-unit')
 
 module.exports = function (id, type) {
   var orgUnit = orgUnitFinder('name', type)
 
   var table = orgUnit.capacityBreakdownView
-  var whereObject = {}
 
-  if (id !== undefined) {
-    whereObject.id = id
+  var selectList = [
+    'total_points AS totalPoints',
+    'available_points AS availablePoints',
+    'reduction_hours AS reductionHours',
+    'link_id AS linkId',
+    'grade_code AS grade',
+    'total_cases AS totalCases',
+    'cms_adjustment_points AS cmsAdjustmentPoints',
+    'gs_adjustment_points AS gsAdjustmentPoints',
+    'arms_total_cases AS armsTotalCases',
+    'contracted_hours AS contractedHours'
+  ]
+
+  var requiresWorkloadOwnerName = (type === ORGANISATION_UNIT.TEAM.name)
+
+  if (requiresWorkloadOwnerName) {
+    selectList.push('CONCAT(forename, \' \', surname) AS name')
```

Comment:
I think we should be just returning the forename and surname as fields and let the user of this call handle the  concatenation